weave/mongo_loader.py

```diff
@@ -179,3 +178,35 @@ def load_mongo(self, uuids, **kwargs):
         self.load_mongo_metadata(uuids, collection=metadata_collection)
         self.load_mongo_manifest(uuids, collection=manifest_collection)
         self.load_mongo_supplement(uuids, collection=supplement_collection)
+
+
+    def remove_document (self, uuid : str, **kwargs):
+        """Delete a document using the uuid in the supplement,
```

Comment:
Per the weave style guide, the first line of the docstring should be a brief description, followed by a new line, followed by a more detailed description. It looks like here you have the second sentence as a continuation of the first. I think this should be reworked according to the weave style guide and other docstrings in this repo.

Reply:
fixed